src/hdusd/ui/render.py

```diff
@@ -13,11 +13,45 @@
 # limitations under the License.
 #********************************************************************
 import bpy
+import webbrowser
 
-from . import HdUSD_Panel
+from . import HdUSD_Panel, HdUSD_Operator
+
+from hdusd import bl_info
 
 from pxr import UsdAppUtils, Tf
 
+from rpr import RprUsd
+
+class HDUSD_OP_open_web_page(HdUSD_Operator):
+    """
+    Operator to open web pages. Available page types:
+    - 'main_site'
+    - 'documentation'
+    - 'downloads'
+    - 'community'
+    - 'bug_reports'
+    """
+
+    bl_idname = "hdusd.op_open_web_page"
+    bl_label = "Open Web Page"
+    bl_description = "Open web page in browser"
+
+    page: bpy.props.StringProperty(name="Page")
+
+    def execute(self, context):
+        url = {
+            'main_site':     "https://www.amd.com/en/technologies/radeon-prorender",
+            'documentation': "https://radeon-pro.github.io/RadeonProRenderDocs/en/usd_hydra/about.html",
+            'downloads':     "https://www.amd.com/en/technologies/radeon-prorender-downloads",
+            'community':     "https://community.amd.com/t5/blender-discussions/bd-p/blender-discussions",
```

Comment:
I think we should point users to https://github.com/GPUOpen-LibrariesAndSDKs/BlenderUSDHydraAddon/discussions